pkg/api/v1/schema.go

```diff
@@ -94,6 +97,23 @@ type CPU struct {
 	Cores uint32 `json:"cores,omitempty"`
 }
 
+// Memory allow specifying the VirtualMachineInstance memory features
+// ---
+// +k8s:openapi-gen=true
+type Memory struct {
+	// Hugepages allow to use hugepages for the VirtualMachineInstance instead of regular memory.
+	// +optional
+	Hugepages *Hugepages `json:"hugepages,omitempty"`
+}
+
+// Hugepages allow to use hugepages for the VirtualMachineInstance instead of regular memory.
+// ---
+// +k8s:openapi-gen=true
+type Hugepages struct {
+	// PageSize specifies the hugepage size.
+	PageSize string `json:"pageSize,omitempty"`
```

Comment:
Can you add something like `for instance for x86_64 valid values are 1Gi and 2Mi`. To give a hint of the formatting and possible values.

Reply:
done